src/network/networkd-link.h

```diff
@@ -52,6 +52,7 @@ typedef struct Link {
         int ifindex;
         int master_ifindex;
         int dsa_master_ifindex;
+        int phys_port_ifindex; /* for SR-IOV */
```

Comment:
Maybe add a comment here that this is initially 0, and will later be set to either -1 or positive?

Reply:
Sure, will do so after it is confirmed that this PR works well.